### DISCUSSION TITLE
Would you please remove "sdwebui" from the name of the repository

### DISCUSSION THREAD

**Discussion by gilroff** [ljleb/sd-webui-neutral-prompt#36] [Ideas]
That would not terminate free colab when using your extension.

**Comment by ljleb** [comment]
Does renaming the directory of the extension in your local installation work for you? It is common practice to prefix webui extensions with "sd-webui".

**Reply by gilroff** [reply]
Its okay no need to change. I forked your extension and removed the sdwebui from name. it works!
Btw thanks for the awesome extension.

**Reply by ljleb** [reply]
You can clone an extension with `git clone <extension-git-url> <folder-name>` and it should clone the repository to "folder-name". IIUC this should do the trick, unless colab looks into .git/ objects or executed clone commands.

**Reply by gilroff** [reply]
oh thanks I will do that.
